python/aswfdocker/migrater.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import subprocess
+import typing
+
+from aswfdocker import constants, utils
+
+logger = logging.getLogger(__name__)
+
+
+class MigrateInfo:
+    def __init__(self, image, version, source, destination):
+        self.image = image
+        self.version = version
+        self.source = source
+        self.destination = destination
+
+
+class Migrater:
+    def __init__(self, from_org: str, to_org: str):
+        self.from_org = from_org
+        self.to_org = to_org
+        self.migration_list: typing.List[MigrateInfo] = []
+        self.cmds: typing.List[str] = []
+
+    def gather(self, package: str, version: str):
+        for pkg, versions in constants.VERSIONS[constants.ImageType.PACKAGE].items():
+            image = "ci-package-" + pkg
+            if not package or package == pkg:
+                for v in versions:
+                    major_version = v.split(".")[0]
+                    if not version or version == major_version:
+                        self.migration_list.append(
+                            MigrateInfo(
+                                image,
+                                v,
+                                f"docker.io/{self.from_org}/{image}:{v}",
```

Comment:
Rather than hardcoding the path to the docker container repository, would it make sense to make this configurable? We might move to a different container registry at some point, possibly leveraging GitHub Packages for instance.